can changes go back to incubaid?

despiegk opened the pull request:
or do I make a mistake here?

domsj commented:
I think you can do as you please ... the incubaid repo is all yours.
Do note that the license has changed from apache 2 to agpl.
And you're merging our 1.9 branch into a 1.8 branch on this repo.

Commit:
make it work with git

Commit:
Merge pull request #115 from openvstorage/setup.py2git

make it work with git

Commit:
add cmx files in installed client

Commit:
Merge pull request #117 from openvstorage/cmx

add cmx files in installed client

domsj committed:
sigh centos

domsj committed:
Merge pull request #119 from openvstorage/centos-sigh

sigh centos

domsj committed:
disable travis cache as it currently makes the build fail due to a lack of space

Commit:
assert serialization of counted lists

domsj committed:
Merge pull request #120 from openvstorage/travis

disable travis cache as it currently makes the build fail due to a la…

Commit:
Merge pull request #121 from openvstorage/assert_counted_list_ser

assert serialization of counted lists

domsj committed:
introduce client config ini format, which includes tls info

domsj committed:
allow getting arakoon config from another arakoon

domsj committed:
update arakoon_client.mllib

domsj committed:
implement to_ini for arakoon_client_config

Commit:
fix config bug tlog_max_size

HarukaMa committed:
fix build

Commit:
autofix for missing .tlog file and (failing) test

domsj committed:
bugfix

Commit:
bug fix: correct iterate for tiny tlogs

Commit:
remove cucu

domsj committed:
Merge pull request #124 from openvstorage/client-config

Expanded arakoon client config + allow getting config from another arakoon

Commit:
Merge branch '1.9' into issue_125

Commit:
conditionally remove database

Commit:
Merge pull request #127 from openvstorage/issue_125

Issue 125

domsj committed:
skip leading slash in arakoon config path

Commit:
verify_store;get-raw; set < file

domsj committed:
Merge pull request #129 from openvstorage/arakoon-config-path

skip leading slash in arakoon config path

Commit:
Merge pull request #130 from openvstorage/verify_db

verify_store;get-raw; set < file

Commit:
removed 'down', loop when not canceled

Commit:
Merge pull request #131 from openvstorage/issue_126

removed 'down', loop when not canceled

Commit:
make_release: 1.9.10

Commit:
Merge pull request #132 from openvstorage/1.9.10

make_release: 1.9.10

Commit:
fix race

make sure reinit and remove_below keep out of each other's way.

Commit:
2 more tests for collapse

Commit:
align ubuntu starting image

CI will eat less

Commit:
Merge pull request #134 from openvstorage/issue_133

Issue 133

Commit:
make_release: 1.9.11

Commit:
master can have more valid states

Commit:
Merge pull request #136 from openvstorage/ci_fixes

master can have more valid states

Commit:
Merge pull request #135 from openvstorage/1.9.11

make_release: 1.9.11

Commit:
test for scenario

domsj committed:
load plugins for local collapse

domsj committed:
Merge pull request #141 from openvstorage/local-collapse-load-plugins

load plugins for local collapse

domsj committed:
make_release: 1.9.12

domsj committed:
Merge pull request #142 from openvstorage/1.9.12

make_release: 1.9.12

domsj committed:
update ocaml-redis

domsj committed:
Merge pull request #143 from openvstorage/update-ocaml-redis

update ocaml-redis

Commit:
make_release: 1.9.13

Commit:
Merge pull request #144 from openvstorage/1.9.13

make_release: 1.9.13

Commit:
failing test for learner case

Commit:
Merge pull request #145 from openvstorage/failing_learner

failing test for learner case

Commit:
control where tests live

Commit:
add a 'nose' suite

Commit:
Merge pull request #146 from openvstorage/jenkins_test_home

Jenkins test home

Commit:
which pgrep?

Commit:
Merge pull request #147 from openvstorage/more_jenkins

verbose output; which arakoons are running?

Commit:
Merge branch '1.9' into issue_93

Commit:
is fuser a better option?

Commit:
Popen + wait

& no naked subprocess usage

Commit:
does python matter?

Commit:
avoid -daemonize

Commit:
avoid -daemonize

domsj committed:
bash as PID 1 in container

Commit:
Merge branch 'issue_93-jan' into issue_93

Commit:
zombie reaper for 1604

Commit:
inject via Popen

Commit:
more robust etcd test

Commit:
stability

Commit:
Merge pull request #148 from openvstorage/issue_93

test for scenario (robustify tests, unshadow bugs)

Commit:
invalidate tlog map after catchup_tlog

Commit:
only add to compression after entry is written

Commit:
Merge pull request #150 from openvstorage/issue_137

invalidate tlog map after catchup_tlog

Commit:
Merge pull request #151 from openvstorage/tlog_absence

only add to compression after entry is written

Commit:
move to camltc-0.9.4

Commit:
make_release: 1.9.14

Commit:
Merge pull request #153 from openvstorage/1.9.14

make_release: 1.9.14

Commit:
route through "$@"

Commit:
Merge pull request #154 from openvstorage/centos_build_fix

route through "$@"

Commit:
Merge pull request #152 from openvstorage/camltc-0.9.4

move to camltc-0.9.4

domsj committed:
make_release: 1.9.15

domsj committed:
Merge pull request #155 from openvstorage/1.9.15

make_release: 1.9.15

domsj committed:
fix centos build (new epel release)

domsj committed:
Merge pull request #158 from openvstorage/centos

Fix centos build (new epel release)

Commit:
Update Index page of Arakoon Gitbook https://github.com/openvstorage/… (#156)

* Update Index page of Arakoon Gitbook https://github.com/openvstorage/ovs-documentation/issues/54

* Remove "Where is Arakoon used in Open vStorage" section

* Review comments on PR

Commit:
fix docker (by moving to core version that still works)

- remove use of Core.Time_internal & move to 114.35+03
- update versions of other libs too

Commit:
closer to orig

Commit:
comments

Commit:
Merge pull request #159 from openvstorage/docker_fix

fix docker (by moving to core version that still works)

Commit:
dump synced sequence in full splendor too

domsj committed:
add inspect-store command

domsj committed:
Merge pull request #163 from openvstorage/inspect-tc

add inspect-store command

Commit:
Merge pull request #162 from openvstorage/SyncedSeq2s

dump synced sequence in full splendor too

Commit:
64KB Lwt_io.default_buffer_size

Commit:
1MB buffer for file copy

Commit:
fix bug in copy_file

Commit:
show problems in travis

Commit:
Merge pull request #164 from openvstorage/default_buffer_size

64KB Lwt_io.default_buffer_size

Commit:
make_release: 1.9.16

Commit:
Merge pull request #165 from openvstorage/1.9.16

make_release: 1.9.16

Commit:
test can be run twice in a row now

Commit:
Merge pull request #166 from openvstorage/test_catchup_mixed_config

test can be run twice in a row now

domsj committed:
cleanup tlogs after local collapse, even when there was nothing to collapse

domsj committed:
Merge pull request #169 from openvstorage/collapse-local-cleanup

cleanup tlogs after local collapse, even when there was nothing to co…

domsj committed:
check cluster id in store, to prevent ops errors

domsj committed:
PR comment (use cluster_id from config for local collapse)

domsj committed:
Merge pull request #170 from openvstorage/store-check-cluster-id

check cluster id in store, to prevent ops errors

Commit:
add tlogs_to_keep parameter to cli

remove warnings

Commit:
make_release: 1.9.17

Commit:
move to v.0.9.114.41+37

because they broke the one we were using.

Commit:
Merge pull request #173 from openvstorage/change_janestreet_repo

move to v.0.9.114.41+37

Commit:
Merge pull request #171 from openvstorage/issue_168

add tlogs_to_keep parameter to cli

Commit:
Merge pull request #172 from openvstorage/1.9.17

make_release: 1.9.17

Commit:
changes for Lwt.3.0.0

Commit:
Lwt_unix.bind signature change

Commit:
Merge pull request #174 from openvstorage/lwt.3.0.0

Lwt.3.0.0

Commit:
no longer depend on janestreet's opam repo

Commit:
Merge pull request #175 from openvstorage/rejoin_main_opam

no longer depend on janestreet's opam repo

Commit:
make_release: 1.9.18

Commit:
Merge pull request #176 from openvstorage/1.9.18

make_release: 1.9.18

JeffreyDevloo committed:
Updated pyrakoon

domsj committed:
Merge pull request #178 from openvstorage/update_pyrakoon

Updated pyrakoon

Commit:
failing tests

Commit:
fix

Commit:
extra test (and fix the one we had)

Commit:
Merge pull request #184 from openvstorage/find_start_bug

Find start bug

domsj committed:
allow fiddling with store i

Commit:
make_release: 1.9.19

domsj committed:
Merge pull request #186 from openvstorage/fiddle-store-i

allow fiddling with store i

Commit:
Merge pull request #187 from openvstorage/1.9.19

make_release: 1.9.19

Commit:
logging tweaks

- more fine grained configuration of connection logging
- replace ~exn with ": %S" (Printexc.to_string exn)

Commit:
Merge pull request #188 from openvstorage/log_level_tweaks

logging tweaks

Commit:
harvest_limited

domsj committed:
change a few log levels from info to debug

domsj committed:
throttling for optimize db

domsj committed:
optimize File_system.copy_file to avoid 2 userspace copies

domsj committed:
Merge pull request #192 from openvstorage/optimize-db-slowdown

throttling for optimize db

domsj committed:
Merge pull request #193 from openvstorage/optimize-copy-file

optimize File_system.copy_file to avoid 2 userspace copies

Commit:
cater for value & message when harvesting

Commit:
recycle send buffer for msg serialization

Commit:
make_release: 1.9.20

Commit:
Merge pull request #190 from openvstorage/harvest_limited

harvest_limited

Commit:
upgrade dependencies

Commit:
Merge pull request #195 from openvstorage/1.9.20

make_release: 1.9.20

Commit:
typo

Commit:
big sequences can flood logs: no more.

Commit:
Merge pull request #197 from openvstorage/1.9.19.no_big_dump

big sequences can flood logs: no more.

Commit:
add update.mli

Commit:
Merge pull request #198 from openvstorage/update.mli

add update.mli

Commit:
make_release: 1.9.21

Commit:
Merge pull request #199 from openvstorage/1.9.21

make_release: 1.9.21

Commit:
explicit option for ppx_deriving.4.1 & base.v0.9.3

Commit:
Merge pull request #200 from openvstorage/ocaml.4.04.2

Ocaml.4.04.2

Commit:
back to Lwt.3.0.0

TLS tests fail with 3.1.0: fe
server.quick.system_tests_ssl.TestTLS.test_inter_node_communication

Commit:
Merge pull request #201 from openvstorage/lwt.3.0.0

back to Lwt.3.0.0

Commit:
make_release: 1.9.22

Commit:
Merge pull request #202 from openvstorage/1.9.22

make_release: 1.9.22

Commit:
Upgrade toolchain, opamify build

delamonpansie committed:
Merge pull request #206 from openvstorage/toolchain-upgrade

Upgrade toolchain, opamify build

Commit:
make_release: 1.9.23

Commit:
Merge pull request #207 from openvstorage/1.9.23

make_release: 1.9.23

Commit:
build arakoon_version.ml with -opaque

Commit:
needed opam switch inside the repo

Commit:
Merge pull request #208 from openvstorage/opaque-version

build arakoon_version.ml with -opaque

Commit:
remove ubuntu.14.04 & centos-7

Commit:
Dynamic JOBS calculation

Commit:
Merge pull request #209 from openvstorage/opam_switch

needed opam switch inside the repo

Commit:
Silence nonrelevant error

Commit:
Pass -use-ocamlfind implicitly

Commit:
Build arakoon.native before testing

Commit:
Remove unused read_only

Commit:
Do not polute $HOME

Commit:
Decrease op_count to sane value

Commit:
Indent

Commit:
Add comments

Commit:
Simplify drop_master

Commit:
Factor out decline_client_update

Commit:
Simplify decline_client_update

Commit:
Split slave_steady_state

domsj committed:
allow loading a plugin when using --replay-tlogs so userfunctions can be available

domsj committed:
pr comment

Commit:
Learner node

domsj committed:
Merge pull request #211 from openvstorage/replay-tlogs-load-plugin

allow loading a plugin when using --replay-tlogs so userfunctions can be available

Commit:
Minor cleanup

delamonpansie committed:
Merge pull request #213 from openvstorage/readability

Minor cleanup

Commit:
Remove dead code.

Condition current_i <= future_i is always true.

Commit:
Indent

Commit:
Quote binary strings

delamonpansie committed:
Merge pull request #215 from openvstorage/quote-binary-string

Quote binary strings

Commit:
add tzdata package

Commit:
clean builds

Commit:
Remove unused script

Commit:
Fix: under "set -u" variable access must be guarded against undefined

Commit:
Move ./run_with_timeout_and_progress.sh into travis.sh

Commit:
Remove unused depends

Commit:
Merge pull request #216 from openvstorage/tzdata

Tzdata

Commit:
Fix progress printing broken in f46623f

delamonpansie committed:
Merge pull request #217 from openvstorage/cleanup

Cleanup

delamonpansie committed:
Merge pull request #214 from openvstorage/dead-code

Remove dead code.

Commit:
Lock TZ at startup

This will save one stat(/etc/localtime) syscall per log print

delamonpansie committed:
Merge pull request #219 from openvstorage/no-stat-localtime

Lock TZ at startup

delamonpansie committed:
Merge pull request #210 from openvstorage/learner-node

Learner node

domsj committed:
add inspect-cluster command

domsj committed:
fix issue in tlog_map when last part of a catchup is a .tlx

domsj committed:
'unit' (ocaml) test suite should make clean first too

domsj committed:
fix failing tests

domsj committed:
expand inspect-cluster command a bit more

domsj committed:
bugfix: failed catchup might apply one entry too far to the store

domsj committed:
Merge pull request #220 from openvstorage/inspect-cluster

add --inspect-cluster command

Commit:
make_release: 1.9.24 (#221)

Commit:
large dependency upgrade + fix more packages (#222)

Commit:
move to ezxmlm.1.0.2 (#224)

Commit:
make_release: 1.9.25 (#223)

Commit:
typo

andersfugmann committed:
Prefix inifiles library with arakoon to avoid conflict with ocaml-inifiles

andersfugmann committed:
Merge pull request #225 from openvstorage/wrap_inifiles

Prefix inifiles library with arakoon to avoid conflict with ocaml-inifiles

Commit:
opam's snappy no longer likes 4.06

Commit:
Merge pull request #226 from openvstorage/snappy_anti_revisionism

opam's snappy no longer likes 4.06

andersfugmann committed:
Rename interval.ml to arakoon_interval.ml

andersfugmann committed:
Merge pull request #227 from openvstorage/fix_module_name_clash

Rename interval.ml to arakoon_interval.ml

andersfugmann committed:
Add core to the list of dependencies for arakoon client

andersfugmann committed:
Merge pull request #228 from openvstorage/update_meta_dependencies

Add core to the list of dependencies for arakoon client

Commit:
allow DeletePrefix in a Sequence

Commit:
Merge pull request #231 from openvstorage/DeletePrefix_in_sequence

allow DeletePrefix in a Sequence

Commit:
allows catchup-only to pick a specific node

Commit:
vendoring camltc package

Commit:
sourceNode parameter in cluster mgmt

Commit:
Merge pull request #233 from openvstorage/specific_catchup_only

Specific catchup only

Commit:
deprecate fsync, replace with fsync_interval and fsync_entries

Commit:
Merge pull request #234 from openvstorage/fsync_knobs

deprecate fsync, replace with fsync_interval and fsync_entries

JeffreyDevloo committed:
Update pyrakoon to the refactored branch

Commit:
Merge pull request #235 from openvstorage/1.9_update_submodule

Update pyrakoon to the refactored branch

Commit:
add test for inspect

Commit:
move inspect to common to make it usable outside of test

Commit:
remove compiler warning

Commit:
restoring dump_store in all its optional grandor

Commit:
restart_master_long times 3

Commit:
jenkins can't handle it

Commit:
Merge pull request #236 from openvstorage/test_inspect

Test inspect

Commit:
instead of competing, let the other racing slave win

Commit:
latest pyrakoon

Commit:
Merge pull request #237 from openvstorage/multi_paxos_improvement

instead of competing, let the other racing slave win

Commit:
move to 4.07.1

Commit:
adds AssertRange to python client

toolslive committed:
Merge pull request #238 from openvstorage/python_assert_range

Python assert range

toolslive committed:
Merge pull request #239 from openvstorage/4.07.1

4.07.1

Commit:
smoother jenkins experience (no more out of memory)

Commit:
clean $TEST_HOME/tmp

Commit:
removed Jan Doms as author (upon his request)

Commit:
Merge branch '1.9' of github.com:openvstorage/arakoon into 1.9

Commit:
Merge pull request #62 from openvstorage/fix_jenkins

smoother jenkins experience (no more out of memory)
